shared/js/ui/pages/options.es6.js

```diff
@@ -48,12 +50,12 @@ Options.prototype = window.$.extend({},
     },
 
     generateFeedbackLink: function () {
-      const mailto = `mailto:extension-feedback@duckduckgo.com?subject=${this.isBrowser}%20Extension%20Feedback&body=Help%20us%20improve%20by%20sharing%20a%20little%20info%20about%20the%20issue%20you%27ve%20encountered%2E%0A%0ATell%20us%20which%20features%20or%20functionality%20your%20feedback%20refers%20to%2E%20What%20do%20you%20love%3F%20What%20isn%27t%20working%3F%20How%20could%20it%20be%20improved%3F%20%20%0A%0A`
```

Comment:
Switched these over to use the templates because I'm a lazy person.

Reply:
Thanks for this!

Reply:
nice. The mailto string should probably be generated, too..  at some point in the future.